coderd/database/queries/workspaces.sql

```diff
@@ -44,22 +44,39 @@ FROM
 	workspaces
 LEFT JOIN LATERAL (
 	SELECT
+		build_number,
+		workspace_builds.job_id,
 		workspace_builds.transition,
 		provisioner_jobs.started_at,
 		provisioner_jobs.updated_at,
 		provisioner_jobs.canceled_at,
 		provisioner_jobs.completed_at,
-		provisioner_jobs.error
+		provisioner_jobs.error,
+		workspace_agents.created_at,
+		workspace_agents.disconnected_at,
+		workspace_agents.first_connected_at,
+		workspace_agents.last_connected_at,
+		workspace_agents.connection_timeout_seconds
```

Comment:
Suggestion: renaming these, e.g. like `workspace_agents.created_at AS agent_created_at` would make the rest of the query more clear. I was a bit confused at first when reading `latest_build.created_at`, etc.

Alternatively: These could also precompute the agent states, like `(first_connect IS NULL AND ...) AS agent_timeout`

Reply:
Done